src/parameterized_tendencies/microphysics/microphysics_wrappers.jl

```diff
@@ -3,22 +3,42 @@
 import Thermodynamics as TD
 import CloudMicrophysics.Microphysics0M as CM0
 
+# define some aliases and functions to make the code more readable
+const Iₗ = TD.internal_energy_liquid
+const Iᵢ = TD.internal_energy_ice
+const qₜ = TD.PhasePartition
```

Comment:
I think q_t is a bit confusing, but I don't have a better name. q_pt may be better, but we are using q_pt in some other places in the code so I'm not sure if that's less confusing.

Reply:
How about `PP = TD.PhasePartition` ?